package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-pay-processing",
-  "version": "2.20.1",
+  "version": "2.20.2",
```

Comment:
Would bump the minor version rather than patch -- change of payload will effect previous and future deploys